assets/js/index.js

```diff
@@ -0,0 +1,914 @@
+					this.scrollLock = true;
+				}.bind(this), 250);
+			}
+		}
+
+		// Check for modal matching querystring when clicking a link
+		// Format: www.url.com?modal=modal-id
+		document.querySelectorAll('a').forEach(function (link) {
+			link.addEventListener('click', function (event) {
+				// Load based on query string
+				if (event.target.href && event.target.href.indexOf(key) !== -1) {
+					var modalTargetString = twentytwentyGetQueryStringValue(key, event.target.attr('href')),
+						modalTarget = document.querySelector('#' + modalTargetString);
+
+					if (modalTargetString && modalTarget.length) {
+						modalTarget.classList.add('active').triggerHandler('toggled');
+						this.scrollLock = true;
+
+						return false;
+					}
+				}
+			}.bind(this));
+		});
+	},
+
+	// Hide and show modals before and after their animations have played out
+	hideAndShowModals: function () {
+		var modals = document.querySelectorAll('.cover-modal');
+
+		// Show the modal
+		modals.forEach(function (modal) {
+			modal.addEventListener('toggle-target-before-inactive', function (event) {
+				if (event.target !== modal) {
+					return;
+				}
+
+				modal.classList.add('show-modal');
+			});
+
+			// Hide the modal after a delay, so animations have time to play out
+			modal.addEventListener('toggle-target-after-inactive', function (event) {
+				if (event.target !== modal) {
+					return;
+				}
+
+				setTimeout(function () {
+					modal.classList.remove('show-modal');
+				}, 500);
+			});
+		});
+	},
+
+	// Untoggle a modal
+	untoggleModal: function (modal) {
+		var modalToggle = false;
+
+		// If the modal has specified the string (ID or class) used by toggles to target it, untoggle the toggles with that target string
+		// The modal-target-string must match the string toggles use to target the modal
+		if (modal.dataset.modalTargetString) {
+			var modalTargetClass = modal.dataset.modalTargetString;
+			modalToggle = document.querySelector('*[data-toggle-target="' + modalTargetClass + '"]');
+		}
+
+		// If a modal toggle exists, trigger it so all of the toggle options are included
+		if (modalToggle) {
+			modalToggle.click();
+
+			// If one doesn't exist, just hide the modal
+		} else {
+			modal.classList.remove('active');
+		}
+	},
+
+}; // twentytwenty.coverModals
+
+/*	-----------------------------------------------------------------------------------------------
+	Dynamic Screen Height
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.dynamicScreenHeight = {
+
+	init: function () {
+		this.setScreenHeight();
+
+		setTimeout(function () {
+			this.setScreenHeight();
+		}.bind(this), 500);
+
+		window.addEventListener('resize', function () {
+			this.setScreenHeight();
+		}.bind(this));
+	},
+
+	setScreenHeight: function () {
+		var screenHeight = document.querySelector('.screen-height');
+		if (screenHeight) {
+			screenHeight.style.minHeight = window.innerHeight;
```

Comment:
This doesn't work, it should be `window.innerHeight + 'px'`

Also, setTimeout has any purpose in `init()`?